docs/build/configure/_index.md

```diff
@@ -227,17 +227,22 @@ Find more information in our [remotes documentation](/build/configure/parts-and-
 
 To automatically run a specified script when the machine boots, configure a _process_.
 
-Start by giving the process a **Name** (`id` in raw JSON) (an identifier of your choice) and clicking **Create Process**.
+Start by giving the process a **Name** (`id` in raw JSON) (an identifier of your choice) and clicking **Create process**.
 
 Then fill in the following fields:
 
 - **Executable** (`name`): The command you want to execute when your machine connects to the server.
 - **Arguments** (`args`): Optional arguments to follow the command.
-- **Working Directory** (`cwd`): Where you want the process to execute.
+- **Working directory** (`cwd`): Where you want the process to execute.
   An optional setting that defaults to the directory where `viam-server` executes.
-
-You can also toggle whether you want errors and other messages to be logged, and whether to execute the command just once or keep running the process indefinitely.
-In raw JSON, these options are represented by `log` (bool) and `one_shot` (bool), respectively.
+- **Logging** (`log`): Toggle logging of errors and other messages on or off (boolean).
```

Comment:
I think the (`boolean`) at the end is odd and not consistent with the other list items which dont have `(string)`. Would a table be more suitable and consistent with how we generally document config fields?
```suggestion
- **Logging** (`log`): Toggle logging of errors and other messages on or off.
```

Reply:
I can try a table and see how it looks. I think this was originally written with a more how-to style in mind, sort of consistent with how we write tutorials with unordered lists, or how https://docs.viam.com/data/capture/#add-the-data-management-service uses steps and sentences. This one is also weird in that the builder labels don't match the raw JSON at all.